webdiff/app.py

```diff
@@ -279,11 +282,19 @@ def run():
 
     PORT = pick_a_port(parsed_args)
 
-    sys.stderr.write('''Serving diffs on http://localhost:%s
+    if app.config.get('USE_HOSTNAME'):
+        _hostname = platform.node()
+        # platform.node will return empty string if it can't find the hostname
+        if not _hostname:
+            sys.stderr.write('Warning: hostname could not be determined')
+        else:
+            HOSTNAME = _hostname
+
+    sys.stderr.write('''Serving diffs on http://%s:%s
 Close the browser tab or hit Ctrl-C when you're done.
-''' % PORT)
+''' % (HOSTNAME, PORT))
     Timer(0.1, open_browser).start()
-    app.run(host='0.0.0.0', port=PORT)
+    app.run(host=HOSTNAME, port=PORT)
```

Comment:
We haven't run into any issues because we've been working with a locally modified version (essentially my patch in this branch), that sets the webdiff to use the clients hostname for the flask server.

How would you like to proceed? Keep it as 'localhost' and essentially have it be private unless you set the config to use hostnames? Or use a conditional to check if ```HOSTNAME=='localhost'``` in which case we'll start the server on ```0.0.0.0```?